apps/patient/src/components/PharmacyInfo.tsx

```diff
@@ -90,9 +89,7 @@ export const PharmacyInfo = ({
   preferred = false,
   showDetails = true,
   boldPharmacyName = true,
-  isStatus = false,
-  price = null,
-  selected = false
```

Comment:
Do we not need the selected?

Reply:
it was only being used to surface the cash price disclaimer, now that that's getting removed, there's no reason for that prop